lib/core/filtering/simple_rule.rb

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+module Core
+  module Filtering
+    class SimpleRule
+      POLICY = 'policy'
+      FIELD = 'field'
+      RULE = 'rule'
+      VALUE = 'value'
+      ID = 'id'
+
+      DEFAULT_RULE_ID = 'DEFAULT'
+      DEFAULT_RULE = SimpleRule.new(
+        {
+          'policy' => 'include',
+          'field' => '_',
+          'rule' => 'regex',
+          'value' => '.*',
+          'id' => SimpleRule::DEFAULT_RULE_ID
+        }
+      )
+
+      attr_reader :policy, :field, :rule, :value, :id
+
+      def initialize(rule_hash)
+        @policy = rule_hash.fetch(POLICY)
+        @field = rule_hash.fetch(FIELD)
+        @rule = rule_hash.fetch(RULE)
+        @value = rule_hash.fetch(VALUE)
+        @id = rule_hash.fetch(ID)
+      end
+
+      def match?(document)
+        return true if id == DEFAULT_RULE_ID
+        doc_value = document[field]
+        case rule
+        when 'equals'
```

Comment:
Should we perhaps also allow `=` and/or `eq` to mean `equals`? Or is it too confusing?

Reply:
I'd prefer human-readable words over operators. However I wasn't consistent with that when I defined `<` and `>`. @zombieFox @sphilipse do you have strong opinions/designs for this for the UI? I think just aligning the UI and the backend makes sense.

Reply:
I'd propose these options:
1. `equals`, `greater_than` and `less_than`
2. `eq`, `gt` and `lt` (my preference - short and clear)
3. `=`, `>` and `<`
What do you think @seanstory @zombieFox @sphilipse ?